sql/_13_issues/_24_1h/cases/cbrd_25054.sql

```diff
@@ -0,0 +1,141 @@
+drop table if exists test_tbl, test_int, test_short, test_bigint, test_float, test_float_n, test_double, test_numeric, test_numeric_n;
+
+create table test_tbl ( print_ord numeric(3,0) not null );
+insert into test_tbl select rownum *5
+from table({0,1,2,3,4,5,6,7,8,9}) a, table({0,1,2,3,4,5,6,7,8,9}) b, table({0,1,2,3,4,5,6,7,8,9}) c, table({0,1,2,3,4,5,6,7,8,9}) d limit 1000;
+
+select max(print_ord) from test_tbl;
+select rownum, print_ord from test_tbl order by print_ord desc limit 10;
+
```

Comment:
select typeof(-2147483647 - rownum), typeof(2147483646 + rownum);

select typeof(-32767 - rownum), typeof(32766 + rownum);

select typeof(-9223372036854775807 - rownum), typeof(9223372036854775806 + rownum);

select typeof(-3.402823465E+38 - rownum), typeof(3.402823465E+38 + rownum);

select typeof(-1.7976931348623156E+308 - rownum), typeof(1.7976931348623156E+308 + rownum);

select typeof(-999999999999998 - rownum), typeof(999999999999998 + rownum);

Reply:
Reflected the review